server/routes/api/documents/documents.ts

```diff
@@ -1701,4 +1701,48 @@ router.post(
   }
 );
 
+router.post(
+  "documents.empty_trash",
+  auth({ admin: true }),
+  validate(T.DocumentsEmptyTrashSchema),
+  async (ctx: APIContext<T.DocumentsEmptyTrashReq>) => {
+    const { user } = ctx.state.auth;
+    const { ids } = ctx.input.body;
+    const documents = await Document.scope(["withDrafts"]).findAll({
+      where: {
+        id: {
+          [Op.in]: ids,
+        },
+      },
+      paranoid: false,
+    });
+
+    const documentIds = documents.map((document) => document.id);
+    await Document.update(
```

Comment:
If this is needed perhaps it should probably be moved into `documentPermanentDeleter` but I'm not sure it its… presumably you saw an issue?

Reply:
I didn't really.
This is present in the individual document permanent delete API  ([code ref](https://github.com/outline/outline/blob/main/server/routes/api/documents/documents.ts#L1216)); so used it here in case it was needed for a special case.
Is this not needed henceforth?

Reply:
Managed to trace this back to the original implementation (https://github.com/outline/outline/commit/9fccc280d72e91e5cc8efbed73c04095f9be493a#diff-ce4bb28c7d77e61b5a4348fcda078c27194ddb25b68df1d0b2e34fdc3ef26ad9R1189), I think it's needed – but can be moved into the `documentPermanentDeleter` command so that it is shared